src/main/java/com/google/codeu/servlets/UFODataServlet.java

```diff
@@ -0,0 +1,54 @@
+package com.google.codeu.servlets;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonArray;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.util.Scanner;
+
+/**
+ * Returns UFO data as a JSON array, e.g. [{"lat": 38.4404675, "lng": -122.7144313}]
+ */
+@WebServlet("/ufo-data")
+public class UfoDataServlet extends HttpServlet {
+
+    private JsonArray ufoSightingArray;
+
+    @Override
+    public void init() {
+        ufoSightingArray = new JsonArray();
+        Gson gson = new Gson();
+        Scanner scanner = new Scanner(getServletContext().getResourceAsStream("/WEB-INF/ufo-data.csv"));
```

Comment:
Could you explain how this section only takes in longitude and latitude, not the other information separated by commas?

Reply:
Yes, so this code just gets the lat and long coordinates. I had an error and I didnt erase the rest of the data in the CSV file so it just had the lat and long points. I fixed that :)